### PR TITLE
ITSASU-3537: GlobalCYA entire page content update

### PR DESCRIPTION
**Pull request by ymabdulahi**

### CODE REVIEW

File: app/views/agent/GlobalCheckYourAnswers.scala.html [hunk 1]
```diff
@@ -47,65 +51,112 @@
...
             rows = Seq(
+                Some(SummaryListRow(
+                    key = Key(content = Text(messages("agent.global-check-your-answers.income-sources.self-employment.trade"))),
+                    value = Value(content = HtmlContent(s"<strong>${business.trade}</strong>")),
+                    actions = Some(Actions(
+                        items = Seq(
+                            ActionItem(
+                                href = s"${appConfig.agentIncomeTaxSelfEmploymentsFrontendBusinessCheckYourAnswersUrl}?id=${business.id}&isEditMode=true",
+                                content = Text(messages("base.change")),
+                                visuallyHiddenText = Some(messages("agent.global-check-your-answers.income-sources.self-employment.trade"))
```
Comment (ILee15) [inline app/views/agent/GlobalCheckYourAnswers.scala.html:155]: The visually hidden text for changing Sole Trader businesses is "Change Trade" whereas it should specify the trade name, ie "Change {business.trade}"

File: app/views/agent/GlobalCheckYourAnswers.scala.html [hunk 2]
```diff
@@ -47,65 +51,112 @@
...
+
+        @govukSummaryList(SummaryList(
+                rows = Seq(
+                    SummaryListRow(
+                        key = Key(content = Text(messages("agent.global-check-your-answers.income-sources.self-employment.accounting-method"))),
+                        value = Value(content = Text(accountingMethod(soleTraderBusinesses.accountingMethod))),
+                        actions = Some(Actions(
+                            items = Seq(
+                                ActionItem(
+                                    href = s"${appConfig.agentIncomeTaxSelfEmploymentsFrontendBusinessCheckYourAnswersUrl}?id=${soleTraderBusinesses.businesses.head.id}&isEditMode=true",
```
Comment (ILee15) [inline app/views/agent/GlobalCheckYourAnswers.scala.html:129]: &isGlobalEdit=true should be added to the sole trader and property change links

File: test/views/agent/business/GlobalCheckYourAnswersViewSpec.scala [hunk 3]
```diff
@@ -42,339 +42,263 @@ class GlobalCheckYourAnswersViewSpec extends ViewSpec {
...
+    }
+
+    "have a print information link" in {
+      val link = document().mainContent.selectHead("div > p > a.govuk-link")
+      link.text mustBe GlobalCheckYourAnswersMessages.printLink
+      link.attr("data-module") mustBe "hmrc-print-link"
+      link.attr("href") mustBe "#"
+    }
+
     "have an income sources heading" in {
```
Comment (ILee15) [inline test/views/agent/business/GlobalCheckYourAnswersViewSpec.scala:60]: could change this test name to be more appropriate, e.g. "have a check your answers subheading" or something like that

File: app/views/agent/GlobalCheckYourAnswers.scala.html [hunk 4]
```diff
@@ -47,65 +51,112 @@
 @mainTemplate(title = messages("agent.global-check-your-answers.heading"), backLink = Some(backUrl)) {
 
     <h1 class="govuk-heading-l">@messages("agent.global-check-your-answers.heading")</h1>
```
Comment (ILee15) [inline app/views/agent/GlobalCheckYourAnswers.scala.html:53]: The content sheet shows there should be eyebrows caption for agents with the Client name and Nino

File: app/views/agent/GlobalCheckYourAnswers.scala.html [hunk 5]
```diff
@@ -47,65 +51,112 @@
...
-        @correctClientInfo
+         </div>
     </div>
 
+    <h2 class="govuk-heading-m">@messages("agent.global-check-your-answers.subheading")</h2>
+
+    @usingSoftwareSection
+    @taxYearSection(completeDetails.taxYear)
+
+    <h2 class="govuk-heading-m">@messages("agent.global-check-your-answers.income-sources.subheading")</h2>
```
Comment (ILee15) [inline app/views/agent/GlobalCheckYourAnswers.scala.html:70]: The 'Sole trader business' and 'Income from property' H2 subheadings show even when there are no relevant income sources. It doesn't say to hide them, but to me it looks confusing, but maybe worth checking with Design team?

E.g.
Screenshot 2024-12-20 at 15 32 19

**Review by AlexRimmerHMRC — APPROVED** [review]

**Pull request merged**